shared-libs/cht-datasource/src/local/person.ts

```diff
@@ -17,7 +16,7 @@ export namespace v1 {
       return false;
     }
     const hasPersonType = contactTypeUtils.isPerson(settings.getAll(), doc);
-    if (!hasPersonType || !isNormalizedParent(doc)) {
```

Comment:
Decided to go ahead and drop this extra `isNormalizedParent` logic here before this gets released in `4.9` (as originally intended in https://github.com/medic/cht-core/pull/9177/files#r1644945012).